puppetdb_foreman.gemspec

```
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.version     = PuppetdbForeman::VERSION
   s.license     = 'GPL-3.0'
   s.summary     = 'This is a Foreman plugin to interact with PuppetDB.'
-  s.description = 'Disable hosts on PuppetDB after they are deleted or built in Foreman. Follow https://github.com/theforeman/puppetdb_foreman and raise an issue/submit a pull request if you need extra functionality. You can also find some help in #theforeman IRC channel on Freenode.'
+  s.description = 'Disable hosts on PuppetDB after they are deleted or built in Foreman. Follow https://github.com/theforeman/puppetdb_foreman and raise an issue/submit a pull request if you need extra functionality. You can also find some help via the Foreman support pages (https://theforeman.org/support.html).
```

Of course I spot something the moment I press submit.
```suggestion
  s.description = 'Disable hosts on PuppetDB after they are deleted or built in Foreman. Follow https://github.com/theforeman/puppetdb_foreman and raise an issue/submit a pull request if you need extra functionality. You can also find some help via the Foreman support pages (https://theforeman.org/support.html).'
```